2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_setup.py:_flush():71] setting env: {}
2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_init.py:_log_setup():371] Logging user logs to /root/konst/ml-utils/training/tagger_text/wandb/run-20220108_234643-33b3lba0/logs/debug.log
2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_init.py:_log_setup():372] Logging internal logs to /root/konst/ml-utils/training/tagger_text/wandb/run-20220108_234643-33b3lba0/logs/debug-internal.log
2022-01-08 23:46:43,996 INFO    MainThread:618 [wandb_init.py:init():404] calling init triggers
2022-01-08 23:46:43,997 INFO    MainThread:618 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/tags_text/tags_post_texts_v7.csv', 'checkpoint_path': '/root/konst/model_checkpoints/tagger_text', 'model_path': '', 'batch_size': 128, 'epochs': 200, 'start_epoch': 0, 'test_size': 0.2, 'lam': 0, 'patience': 8, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 0}
2022-01-08 23:46:43,997 INFO    MainThread:618 [wandb_init.py:init():460] starting backend
2022-01-08 23:46:43,997 INFO    MainThread:618 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-08 23:46:44,008 INFO    MainThread:618 [backend.py:ensure_launched():216] starting backend process...
2022-01-08 23:46:44,013 INFO    MainThread:618 [backend.py:ensure_launched():222] started backend process with pid: 636
2022-01-08 23:46:44,014 INFO    MainThread:618 [wandb_init.py:init():469] backend started and connected
2022-01-08 23:46:44,018 INFO    MainThread:618 [wandb_init.py:init():533] updated telemetry
2022-01-08 23:46:44,026 INFO    MainThread:618 [wandb_init.py:init():563] communicating current version
2022-01-08 23:46:44,674 INFO    MainThread:618 [wandb_init.py:init():568] got version response 
2022-01-08 23:46:44,674 INFO    MainThread:618 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-08 23:46:44,843 INFO    MainThread:618 [wandb_init.py:init():606] starting run threads in backend
2022-01-08 23:46:45,649 INFO    MainThread:618 [wandb_run.py:_console_start():1810] atexit reg
2022-01-08 23:46:45,650 INFO    MainThread:618 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-08 23:46:45,651 INFO    MainThread:618 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-08 23:46:45,652 INFO    MainThread:618 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-08 23:46:45,652 INFO    MainThread:618 [wandb_init.py:init():633] run started, returning control to user process
2022-01-08 23:47:12,381 INFO    MainThread:618 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-01-08 23:47:12,383 INFO    MainThread:618 [wandb_run.py:_restore():1752] restore
2022-01-08 23:47:14,460 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 35403
  total_bytes: 35403
}

2022-01-08 23:47:14,667 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 35403
  total_bytes: 35403
}

2022-01-08 23:47:14,873 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 35403
  total_bytes: 43157
}

2022-01-08 23:47:14,975 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 35403
  total_bytes: 43157
}

2022-01-08 23:47:15,077 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 36382
  total_bytes: 43157
}

2022-01-08 23:47:15,180 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 43157
  total_bytes: 43157
}

2022-01-08 23:47:15,284 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 43157
  total_bytes: 43157
}

2022-01-08 23:47:15,387 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 43157
  total_bytes: 43157
}

2022-01-08 23:47:15,490 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 43157
  total_bytes: 43157
}

2022-01-08 23:47:16,714 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 43157
  total_bytes: 43157
}

2022-01-08 23:47:17,184 INFO    MainThread:618 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 43157
  total_bytes: 43157
}
local_info {
}

2022-01-08 23:47:18,399 INFO    MainThread:618 [wandb_run.py:_append_files():2180] logging synced files
